src/routes/login/LoginView.vue

```diff
@@ -300,7 +300,7 @@ export default {
       }
     }
   },
-  mounted() {
+  created() {
```

Comment:
This needs to stay mounted because the function is interacting with the DOM (scrolling)
```suggestion
  mounted() {
```

Reply:
Thanks for pointing that out, I have now moved the logic back to mounted so that it runs after DOM is available.